fastlane/lib/fastlane/actions/badge.rb

```diff
@@ -16,7 +16,11 @@ def self.run(params)
           shield_gravity: params[:shield_gravity],
           shield_no_resize: params[:shield_no_resize]
         }
-        Badge::Runner.new.run(params[:path], options)
+        begin
+          Badge::Runner.new.run(params[:path], options)
+        rescue => e
+          UI.user_error!("Something went wrong while running badge: #{e}")
```

Comment:
Why do we catch and then do a user_error? Why don't we re-raise the exception? It would be cool to have some code comments here on this 👍